index.html

```diff
@@ -287,7 +287,7 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		        <div class="container">
 		            <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
-		                    <img alt="Pic" src="img/avatar1.png">
+		                    <img class="col-2" alt="Chahat Bansal" src="img/chahat.png">
```

Comment:
I still don't think this extra class is needed. Can you please explain why you are adding this?